src/main/resources/migration/common/V2__InitialTables.sql

```diff
@@ -0,0 +1,49 @@
+create table alerts
+(
+    alert_id bigserial NOT NULL CONSTRAINT alerts_pk PRIMARY KEY,
+    alert_uuid uuid not null,
+    alert_type varchar(255) not null,
+    alert_code varchar(255) not null,
+    offender_id varchar(255) not null,
+    authorised_by varchar(255) not null,
```

Comment:
May be worth matching the varchar lengths in NOMIS. Type and code are both varchar(12), offender_id is varchar(10) and authorised by is varchar(40). We may choose to support longer authorised by text in future I guess

Reply:
Done.